include/ccl_config.h

```diff
@@ -20,7 +20,8 @@ typedef enum transfer_function_t
   ccl_bbks              = 2,
   ccl_boltzmann_class   = 3,
   ccl_boltzmann_camb    = 4,
-  ccl_boltzmann_isitgr  = 5,
+  ccl_pklin_from_input  = 5,
+  ccl_boltzmann_isitgr  = 6,
```

Comment:
This isn't a big deal but is there a reason not to just add ccl_pklin_from_input on as number 6?

Reply:
I think when I resolved a merge conflict I had already the pklin_from_input set to 5. I have now changed it to 6.